addons/mail/static/src/variables.scss

```diff
@@ -4,6 +4,9 @@
 
 $o-mail-thread-window-zindex: $zindex-modal + 1 !default;
 
+$o-mail-border-radius-sm: .2rem;
+$o-mail-border-radius: .25rem;
+$o-mail-border-radius-lg: .3rem;
```

Comment:
The names are a bit obscure IMO.

Reply:
initial one are $border-radius-sm and so on ... so I don't know what do we expect here

Reply:
Yep... It's just the generic border-radius we use in the context of messaging, with some size variations. Not sure what else we could name.

Reply:
Comments would help understand them better. Not super obvious for me.